wcfsetup/install/files/ts/WoltLabSuite/Core/Form/Builder/Field/RadioButton.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Data handler for a radio button form builder field in an Ajax form.
+ *
+ * @author  Matthias Schmidt
+ * @copyright 2001-2020 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Form/Builder/Field/RadioButton
+ * @since 5.2
+ */
+
+import Field from "./Field";
+import { FormBuilderData } from "../Data";
+import * as Core from "../../../Core";
+
+class RadioButton extends Field {
+  protected _fields: HTMLInputElement[];
+
+  protected _getData(): FormBuilderData {
+    const data = {};
+
+    this._fields.some((input) => {
+      if (input.checked) {
+        data[this._fieldId] = input.value;
+        return true;
+      }
```

Comment:
```suggestion
  protected _getData(): FormBuilderData {
    const data = {};

    this._fields.some((input) => {
      if (input.checked) {
        data[this._fieldId] = input.value;
        return true;
      }
      return false;
```
Be explicit here. I also believe that this is a slight abuse of the `.some()` method :sweat_smile: